client/containers/route/getters.spec.js

```diff
@@ -20,10 +20,11 @@
 // THE SOFTWARE.
 
 import { ROUTE_PARAMS, ROUTE_QUERY } from './getter-types';
-import getters from './getters';
+import getterFns from './getters';
+import { initGetters } from '~test';
```

Comment:
I assume its a follow up on https://github.com/uber/cadence-web/pull/276

Reply:
Follow up to #275